fairworkflows/rdf_wrapper.py

```diff
@@ -309,6 +312,30 @@ def _publish_as_nanopub(self, use_test_server=False, **kwargs):
 
         return publication_info
 
+    @staticmethod
+    def _import_graphviz():
+        """Import graphviz.
+
+        Raises:
+             ImportError with appropriate message if import failed
+        """
+        try:
+            import graphviz
+            return graphviz
+        except ImportError:
+            raise ImportError('Cannot produce visualization of RDF, you need '
+                              'to install graphviz python package. '
+                              'Version 0.14.1 is known to work well.')
+
+    def display_rdf(self):
+        graphviz = self._import_graphviz()
+
+        with TemporaryDirectory() as td:
+            filename = Path(td) / 'dag.dot'
+            with open(filename, 'w') as f:
+                rdf2dot(self._rdf, f)
```

Comment:
Given that `rdf2dot` uses the now deprecated `cgi` stuff that was renamed in newer python versions, could we introduce a version check here to raise an exception (or warning) that the `display_rdf()` method won't work (I think it fails if you try to use `3.8.0` and above. The issue has been fixed, see e.g.

<https://github.com/RDFLib/rdflib/issues/1110>

But the latest release of `rdflib` doesn't have it yet, and they don't seem to release super often...

Reply:
I created a new issue for this: #182